### ISSUE TITLE
The problem when identifying the button is held down

### ISSUE THREAD

**Issue by advice57** [AntiMicroX/antimicrox#859]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe.

When holding a button in the normal way with Antimicro, the letter that that button represents is continuously sent to the notepad. For example, holding down the a button for a while will receive the line aaaaaaaaaa, but if using AntimicroX, you will only get exactly one letter a. Some commands in AutoHotkey that I'm using work very well with Antimicro because the feature of sending out long lines like aaaaaaaaaa makes it easy for AutoHotkey to confirm that the button is being held to trigger an additional script loop to run. If using AntimicroX, AutoHotkey can only run exactly one loop even if you hold the button (here, if you use Toggle or Turbo in Antimicro or AntimicroX, it will cause AutoHotkey to not work effectively).

### Describe the solution you'd like

So I want you to add a feature like AntiMicro's button holding style to AntimicroX.

**Comment by advice57** [comment]
I discovered that I had to select Enable on the repeat key in settings -> general
I can make it press the same as AntiMicro, but when I select Enable then press save and go back to the settings, the "Enable box" is still not selected, so I guess this is an AntiMicroX software error, but in AntiMicro there is no such error. , hope your side will fix it soon.

**Comment by UltraJohn** [comment]
Yep same problem here, the setting is not saved after pressing OK. Unfortunately makes this program unusable for me.
